force-app/main/default/lwc/nksChatAuthenticationInfoV2/nksChatAuthenticationInfoV2.js

```diff
@@ -92,63 +108,57 @@ export default class ChatAuthenticationOverview extends LightningElement {
-                    _this.handleUnsubscribe();
+            if (eventRecordId === this.recordId) {
+                this.currentAuthenticationStatus = response.data.sobject.CRM_Authentication_Status__c;
+                if (this.authenticationComplete) {
+                    if (!this.loginEvtSent) this.sendLoginEvent();
+                    getRecordNotifyChange([{ recordId: this.recordId }]);
+                    this.handleUnsubscribe();
                 }
             }
         };
 
-        subscribe('/topic/Chat_Auth_Status_Changed' /*?Id=" + this.recordId*/, -1, messageCallback).then((response) => {
-            this.subscription = response;
-            console.log('Successfully subscribed to : ', JSON.stringify(response.channel));
-        });
+        subscribe('/topic/Chat_Auth_Status_Changed', -1, messageCallback)
+            .then((response) => {
+                this.subscription = response;
+                console.log('Successfully subscribed to: ', JSON.stringify(response.channel));
+            })
+            .catch((error) => {
+                console.error('Failed to subscribe: ', JSON.stringify(error));
+            });
     }
 
     handleUnsubscribe() {
-        unsubscribe(this.subscription, (response) => {
-            console.log('Unsubscribed: ', JSON.stringify(response));
-        })
-            .then(() => {
-                this.log('Successful unsubscribe');
+        unsubscribe(this.subscription)
+            .then((response) => {
+                console.log('Unsubscribed: ', JSON.stringify(response));
             })
             .catch((error) => {
-                console.log('EMP unsubscribe failed: ' + JSON.stringify(error, null, 2));
+                console.error('EMP unsubscribe failed: ', JSON.stringify(error));
             });
     }
 
     sendLoginEvent() {
-        getCouncellorName({ recordId: this.recordId }).then((data) => {
-            this.councellorName = data;
-            const loginMessage =
-                this.chatLanguage === 'en_US'
-                    ? 'You are now in a secure chat with NAV, you are chatting with ' +
-                      this.councellorName +
-                      '. ' +
-                      CHAT_LOGIN_MSG_EN
-                    : 'Du er nå i en innlogget chat med NAV, du snakker med ' +
-                      this.councellorName +
-                      '. ' +
-                      CHAT_LOGIN_MSG_NO;
-
-            const authenticationCompleteEvt = new CustomEvent('authenticationcomplete', {
-                detail: { loginMessage }
+        getCounselorName({ recordId: this.recordId })
+            .then((data) => {
+                const loginMessage = `${this.labels.initiateMessage} ${data}. ${this.labels.loginMessage}`;
+                const authenticationCompleteEvt = new CustomEvent('authenticationcomplete', {
+                    detail: { loginMessage }
+                });
+                this.dispatchEvent(authenticationCompleteEvt);
```

Comment:
@mamikals tror ikke det er behov for denne lenger.

Reply:
Eventen trengs for å vise login melding til bruker